spec/spec_helper.rb

```diff
@@ -86,6 +86,8 @@
     config.backtrace_exclusion_patterns << %r{/lib\d*/ruby/[0-9]}
     config.backtrace_exclusion_patterns << %r{/gems/[0-9][^/]+/gems/}
   end
+
+  config.after(:suite) { require Rails.root.join("spec/coverage_helper.rb") }
```

Comment:
Should this only be in CI mode?  Does it even make a difference performance wise, particularly if you are running a single spec locally?